README.md

```diff
@@ -58,7 +58,7 @@ See the top of the file for the type definition for what you can configure:
 
 ```ts
 type Config = {
-  /** URL to start the crawl */
+  /** URL to start the crawl, if sitemap is providedm then it will be used instead and download all pages in the sitemap */
```

Comment:
**Suggestion:** Fix the typo in the comment. Replace "providedm" with "provided".
```suggestion
  /** URL to start the crawl, if sitemap is provided then it will be used instead and download all pages in the sitemap */
```